src/plugins/files/server/blob_storage_service/adapters/es/integration_tests/es.test.ts

```diff
@@ -27,10 +27,13 @@ describe('Elasticsearch blob storage', () => {
 
   beforeAll(async () => {
     ElasticsearchBlobStorageClient.configureConcurrentTransfers(Infinity);
-    const { startES, startKibana } = createTestServers({ adjustTimeout: jest.setTimeout });
+
+    const { startES, startKibana } = createTestServers({
+      adjustTimeout: jest.setTimeout,
+    });
     manageES = await startES();
     manageKbn = await startKibana();
-    esClient = manageKbn.coreStart.elasticsearch.client.asInternalUser;
+    esClient = manageKbn.coreStart.elasticsearch.createClient('es.test').asInternalUser;
```

Comment:
__question:__ I'm curious what this change does?

Reply:
This is changing the test to use a dedicated ES Client, instead of a shared one. I added it to squash some flakiness I was experiencing

Reply:
That’s weird, as I can’t think of any plausible reason why there would be an issue with the shared ES client, but I’ll defer to the code owners. Maybe they have some idea.

Reply:
Oh, I see that we set up spies for the ES client methods. Got it, thanks for adding more context @legrego, makes sense now.